go.mod

```diff
@@ -16,7 +17,7 @@ require (
 	github.com/spf13/pflag v1.0.5
 	github.com/submariner-io/admiral v0.16.0-m1
 	github.com/submariner-io/cloud-prepare v0.16.0-m1
-	github.com/submariner-io/lighthouse v0.16.0-m1
+	github.com/submariner-io/lighthouse v0.16.0-m1.0.20230622121749-167ac0da5e51
```

Comment:
This will have to be updated (I know you know, I just want to block the change until we have the right commit hash from merging https://github.com/submariner-io/lighthouse/pull/1280).